server/src/test/java/org/opensearch/discovery/PeerFinderTests.java

```diff
@@ -609,8 +609,8 @@ public void testAddsReachableMasterFromResponse() {
         transportAddressConnector.addReachableNode(discoveredMaster);
         runAllRunnableTasks();
         assertFoundPeers(otherNode, discoveredMaster);
-        assertThat(peerFinder.discoveredMasterNode, nullValue());
-        assertFalse(peerFinder.discoveredMasterTerm.isPresent());
+        assertThat(peerFinder.discoveredClusterManagerNode, nullValue());
+        assertFalse(peerFinder.discoveredClusterManagerTerm.isPresent());
     }
 
     public void testHandlesDiscoveryOfMasterFromResponseFromMaster() {
```

Comment:
`testHandlesDiscoveryOfMasterFromResponseFromMaster ` -> `testHandlesDiscoveryOfClusterManagerFromResponseFromClusterManager `

Reply:
Changed the method name. 👍